opencti-platform/opencti-graphql/src/modules/case/case.graphql

```diff
@@ -132,6 +132,7 @@ interface Case implements BasicObject & StixObject & StixCoreObject & StixDomain
   content: String
   content_mapping: String
   tasks: TaskConnection!
+  objectParticipant: ParticipantConnection
```

Comment:
If it's not for all cases, you can't put it here (because feedback will need to add it too)